2023.05-1week/wintaek/[JS] PRO.120836. 순서쌍의 개수/순서쌍의 개수.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+  const result = [];
+  for (let i = 1; i <= n; i++) {
+    if (n % i === 0) result.push(i);
+  }
+  return result.length;
+}
```

Comment:
```suggestion
function solution(n) {
    let result = 0;
    
    for(let i = 1; i <= n; i++){
        if(n % i === 0) result++;
    }
    
    return result;
}
```

배열에 넣는 법 말고 카운팅 하는 방법은 어떤가요?😮
push 해서 배열에 추가하는 것 보다 카운팅해서 1씩 늘려주는게 효율성면에서 좋지않을까라는 생각을 해봅니당..

Reply:
늦은 시간에 올렸는데도 불구하고 좋은 답변 감사합니다 👍